src/snowflake/cli/api/metrics.py

```diff
@@ -84,24 +85,52 @@ class for holding metrics span data and encapsulating related operations
     START_TIME_KEY: ClassVar[str] = "start_time"
     EXECUTION_TIME_KEY: ClassVar[str] = "execution_time"
     ERROR_KEY: ClassVar[str] = "error"
+    COUNT_SPANS_IN_SUBTREE_KEY: ClassVar[str] = "count_spans_in_subtree"
+    SPAN_DEPTH_KEY: ClassVar[str] = "span_depth"
```

Comment:
can you please document these? Span depth seems to represent the depth of the current span (is it 0-based though? 1-based?). COUNT_SPANS sees to be the total size of the subtree rooted at this span. I would have expected it to be named "SPAN_COUNT" and not "COUNT_SPANS", if so.

Reply:
Updated, tried to make the description as agnostic to the names (e.g. not using the words depth and count) as possible: https://github.com/snowflakedb/snowflake-cli/pull/1835/files#diff-db39ee1dd59a874df561d1420daafd90b948a34c614128415110ee101aa55cc7R88-R92